PostHog/posthog
# feat: console log for session recordings

## Changes

- Adds session console recording
- Upgrades rrweb (had to because console recorder was only added in a newer version) < worth having a look at what this implies for us
- Super simple interface. You can click and the player will scroll to the right spot. Will need some work on the interface, specifically I couldn't get dynamic height rows to work.
- Safe to merge as is, but for the console recording to work we'll need to update posthog-js with this PR

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/1727427/156272380-45ce6558-3f4b-4cd3-af7d-d367c73fe419.png">




*Stay up-to-date with our [coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?



QA: 
- Check out the posthog-js library, run both `yarn` and `python -m SimpleHTTPServer 8080`
- Check out `add-console-log-recording`
- Navigate to http://127.0.0.1:8080/playground/flags/demo.html and click around
- Go to local posthog, add `session-recording-console` feature flag
- Go to recordings and select 'console', see console log coming in
